api/views/file.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from core.models import File
+from etl.tasks import import_file_task
+
+__all__ = ("FileUpdateView",)
+
+logger = logging.getLogger(__name__)
+
+
+class FileUpdateView(APIView):
+    """
+    A view to restart a file.
+    """
+
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        # MVP: We assume this will always be one and only one failed file.
+        qs = File.objects.filter(account=request.data["id"]).filter(
```

Comment:
If we always only want one, I'd suggest using ``first()`` here, e.g.:

```suggestion
        failed_file = File.objects.filter(account=request.data["id"], ...).first()
        if failed_file:
```